mobify/platform-scaffold
# [MBFYCNCT-63] [MBFYCNCT-62] Search products command and search sugges…

This PR covers functionality regarding the searchProducts command and getSearchSuggestions command.

 **JIRA**: ([MBFYCNCT-62](https://thinkwrap.jira.com/browse/MBFYCNCT-62)), ([MBFYCNCT-63](https://thinkwrap.jira.com/browse/MBFYCNCT-63))

## Changes
-  Added searchProducts command
-  Added getSearchSuggestions command 
- Added search to initProductDetailsPage command
-  Added parseSearchSuggestions to app/parsers.js

## Todos
- [ ] Review filters behavior when more than 1 search is done (waiting for confirmation)

## How to test-drive this PR
- Run `npm run dev`
- Preview [Thinkwrap accelerator](https://preview.mobify.com/?url=https%3A%2F%2Fhydemo-electronics.thinkwrap.com%2Fyacceleratorstorefront%2Felectronics%2Fen%2F%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Press the search button in the header
- Enter 'sh' in the search form and click on one of the suggested terms. Should take you to search result page.
- Press the search button in the header
- Enter 'sneakers' in the search form and click on the submit button. Should take you to search result page.
- Verify results correspond to search. Can compare to results obtained here https://hydemo-ukapparel.thinkwrap.com/en/
- Play around with filters, sorting and pagination.
